pkg/internal/checkup/trex/configfiles.go

```diff
@@ -170,3 +170,18 @@ func (c Config) GenerateExecutionScript() string {
 
 	return sb.String()
 }
+
+func GenerateSystemdUnitFile() string {
+	sb := strings.Builder{}
+
+	sb.WriteString("[Unit]\n")
+	sb.WriteString("Description=TRex Server 1\n")
+	sb.WriteString("[Service]\n")
+	sb.WriteString(fmt.Sprintf("WorkingDirectory=%s\n", BinDirectory))
+	sb.WriteString(fmt.Sprintf("ExecStart=%s/%s\n", BinDirectory, ExecutionScriptName))
```

Comment:
suggestion: fmt.Sprintf("ExecStart=%s\n", path.Join(BinDirectory, ExecutionScriptName))

Reply:
this is relevant to other places in this PR where path can be used

Reply:
Done

Reply:
> this is relevant to other places in this PR where path can be used

I'll open a follow-up PR to align all of the commands in `trafficGenBootCommands` to use `path.Join()`